### ISSUE TITLE
How to get the uploaded video url?

### ISSUE THREAD

**Issue by bobananto24** [Ziggeo/ReactNativeSDK#85]
while trying to play the uploaded video from ios the video is not playing. so I have decided to use third party video player. But I am struggling to get the video url. Your suggestions will be more helpful

**Comment by bobananto24** [comment]
@Bane-D  in fact, I couldn't even record the video on iPhone. Till now I have been working in an ios simulator where we cannot record video, so I have tried 
```
import Ziggeo from "react-native-ziggeo-library";
Ziggeo.play("2a2f56f8a0abba209ac9c29bcb3de3aa");
```
then the video is kind of playing in the background
(ie, I could hear the audio but there were no visual changes)
then later when I tried with a real iPhone to record a video, the recorder open for a fraction of a second and closes suddenly
```
async function record(saveId) {
    var appToken = ZIGGEO_APP_TOKEN;
    Ziggeo.setAppToken(appToken);
    Ziggeo.setMaxRecordingDuration(20);
    Ziggeo.setCameraSwitchEnabled(true);
    Ziggeo.setCoverSelectorEnabled(true);
    Ziggeo.setCamera(Ziggeo.FRONT_CAMERA);
    const recorderEmitter = Ziggeo.recorderEmitter();
    const subscription = recorderEmitter.addListener(
      "UploadProgress",
      (progress) =>
        console.log(
          progress.fileName +
            " uploaded " +
            progress.bytesSent +
            " from " +
            progress.totalBytes +
            " total bytes"
        )
    );
    try {
      var token = await Ziggeo.record();
      console.log("Token:" + token);
      if (token) {
        ziggeoSave(token, saveId);
      }
    } catch (e) {
      alert(e);
    }
  } 

```
this is the code I am using for recording

1. NOTE: Everything is working perfectly fine in android
2. another NOTE : pod 'iOS-Client-SDK', :git => 'https://github.com/Ziggeo/iOS-Client-SDK.git', :commit => '295f90fc8777ebe63135359e20fa144409d02e54'
this is the ios client sdk i am using where i couldn't build the app with the latest one
i got this solution from #75 
A solution for this issue will be mre helpful, thanks

**Comment by katolykdev** [comment]
Hi, @bobananto24 
1. You have to install latest install react-native-ziggeo-library.
```"react-native-ziggeo-library": "^1.6.0",```

2. ```Ziggeo.play(VIDEO_TOKEN); ```was deprecated.
Please use ```Ziggeo.playVideo(VIDEO_TOKEN);```

3. ```Ziggeo.record(); ```was deprecated.
Please use ```Ziggeo.startCameraRecorder();```

**Comment by bobananto24** [comment]
thanks @katolykdev but Ziggeo.playVideo(VIDEO_TOKEN) is not working in "ANDROID" (but working in ios simulator)
the error is :
TypeError: ZiggeoPlayer.playVideo is not a function

**Comment by 3akat** [comment]
> Ziggeo.playVideo(VIDEO_TOKEN) is not working in "ANDROID" (but working in ios simulator)
> the error is :
> TypeError: ZiggeoPlayer.playVideo is not a function

fixed in `1.6.3`
